pkg/model/components/kubeproxy.go

```diff
@@ -36,6 +36,10 @@ func (b *KubeProxyOptionsBuilder) BuildOptions(o interface{}) error {
 
 	config := clusterSpec.KubeProxy
 
+	if config.Disable != true {
```

Comment:
If we make this enabled flag you won't have to mess with this. Also I think this is not doing anything anyways.